src/contracts/VotingToChangeKeys.contract.js

```diff
@@ -83,22 +87,8 @@ export default class VotingToChangeKeys {
     return null
   }
 
-  getMiningByVotingKey(_votingKey) {
-    return this.votingToChangeKeysInstance.methods.getMiningByVotingKey(_votingKey).call()
-  }
-
-  async getValidatorActiveBallots(_votingKey) {
-    let miningKey
-    try {
-      miningKey = await this.getMiningByVotingKey(_votingKey)
-    } catch (e) {
-      miningKey = '0x0000000000000000000000000000000000000000'
-    }
-    return await this.votingToChangeKeysInstance.methods.validatorActiveBallots(miningKey).call()
-  }
-
-  async getBallotLimit(_votingKey) {
-    const currentLimit = await this.votingToChangeKeysInstance.methods.getBallotLimitPerValidator().call()
-    return currentLimit - (await this.getValidatorActiveBallots(_votingKey))
+  async getBallotLimit(_miningKey, _limitPerValidator) {
```

Comment:
Function with the same is defined twice: in `VotingToChangeKeys.contract.js` and in `VotingToChangeMinThreshold.contract.js`. Let's add an additional input parameter `methods` and define this function once in helper.

Reply:
@vbaranov There are other functions with identical logic in `*.contract.js` files. Let's solve this in a separate refactoring issue: https://github.com/poanetwork/poa-dapps-voting/issues/167.